### DISCUSSION TITLE
Can we get PermissionEx added to iis:Certificate?

### DISCUSSION THREAD

**Discussion by MikeFarrington** [wixtoolset/issues#7706] [WiX Development]
In converting an installer to v4, and having watched all of Rob's Deployment Dojo video series, I'm inspired to eliminate as many custom actions as possible. 

In [one of the videos](https://www.youtube.com/live/DslwxSg5fPw?si=vaBRaL9iiX2_9PNL&t=727), Rob used PermissionEx apply permissions via SDDL syntax to a folder created with CreateFolder.  I am in need of doing the same to a certificate.  We deploy an SQL AlwaysEncrypted certificate to StoreLocation="localMachine" via iis:Certificate.  However, we need the private key to be exposed to AuthenticatedUsers.  Since Calcs.exe can be used to set permissions on a certificate using SDDL syntax, I was hoping it wasn't too much of a leap to extend PermissionEx to Certificate deployment?

**Comment by robmen** [comment]
You certainly could try. The code that [creates the cert is here](https://github.com/wixtoolset/wix/blob/develop/src/ext/Iis/ca/scacertexec.cpp), and there are a few layers above that to thread any data you need down..
